eng/pipelines/linux.yml

```diff
@@ -28,18 +28,19 @@ stages:
         displayName: Build
         strategy:
           matrix:
-            # Run RedHat6 in release mode on CI to cover Release configuration differences
-            RedHat6_x64_Release:
-              _BuildConfig: Release
-              _architecture: x64
-              _framework: netcoreapp
-              _buildScriptPrefix: ''
-              _buildExtraArguments: /p:RuntimeOS=rhel.6 /p:PortableBuild=false
-              _dockerContainer: rhel6_container
-              _helixQueues: $(redhatHelixQueue)
-              _publishTests: ${{ parameters.fullMatrix }}
+              # Run RedHat7 in release mode on CI to cover Release configuration differences
+              x64_Release:
+                _BuildConfig: Release
+                _architecture: x64
+                _framework: netcoreapp
+                _helixQueues: $(linuxDefaultQueues)
+                _dockerContainer: rhel7_container
+                _buildScriptPrefix: ''
+                _buildExtraArguments: ''
+                _publishTests: true
```

Comment:
The reason why CI is not running is because the indentation is wrong here. You need to remove one tab.

Anyway, this will duplicate Linux x64 in PRs (fullMatrix == false), do we want that?

Reply:
Oops, fixed indent.

Hmm, before CI would have RHEL6 Release + RHEL7 Debug + Alpine36 Debug.
now it has RHEL7 Release + RHEL7 Debug + Alpine36 Debug.

What do you think it should be? Or do I misunderstand what will happen?

Reply:
I think it is fine as you have it.